### ISSUE TITLE
quickactivate "Current project name is nothing"

### ISSUE THREAD

**Issue by behinger** [JuliaDynamics/DrWatson.jl#347]
**Describe the bug**
tldr; I try using quickactivate on a remote cluster, but for some unknown reason, DrWatson.packagename results in "nothing".

```
julia> @quickactivate("LMMPerm")
  Activating project at `/pfs/data5/home/st/XXX/XXX/projects/LMMPerm`
ERROR: The activated project did not match asserted name. Current project name is nothing while the asserted name is LMMPerm.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] quickactivate(path::String, name::String)
   @ DrWatson ~/.julia/packages/DrWatson/S4dYA/src/project_setup.jl:138
 [3] top-level scope
```
Note that the correct environment is activated.

I digged a bit and the reason seems to be:
```
julia> typeof(projectname())
Nothing
```

I looked into the function, and found that 
```julia
julia> Pkg.Types.read_project(Base.active_project())
Pkg.Types.Project(Dict{String, Any}("deps" => Dict{String, Any}("CSV" => "336ed68f-0bac-5ca0-87d4-7b16caf5d00b", "DrWatson" => "634d3b9d-ee7a-5ddf-bec9-22491ea816e1", "StatsModels" => "3eaba693-59b7-5ba5-a881-562e759f1c8d", "GR" => "28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71", "Distributions" => "31c24e10-a181-5473-b8eb-7969acd0382f", "PlutoUI" => "7f904dfe-b85e-4ff6-b463-dae2292396a8", "Random" => "9a3f8284-a2c9-5f02-9a11-845980a1fd5c", "Distributed" => "8ba89e20-285c-5b6f-9357-94700520ee1b", "TimerOutputs" => "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f", "AlgebraOfGraphics" => "cbdf2221-f076-402e-a563-3d30da359d67"…)), nothing, nothing, nothing, nothing, Dict{String, Base.UUID}("CSV" => UUID("336ed68f-0bac-5ca0-87d4-7b16caf5d00b"), "DrWatson" => UUID("634d3b9d-ee7a-5ddf-bec9-22491ea816e1"), "StatsModels" => UUID("3eaba693-59b7-5ba5-a881-562e759f1c8d"), "GR" => UUID("28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71"), "Distributions" => UUID("31c24e10-a181-5473-b8eb-7969acd0382f"), "PlutoUI" => UUID("7f904dfe-b85e-4ff6-b463-dae2292396a8"), "Random" => UUID("9a3f8284-a2c9-5f02-9a11-845980a1fd5c"), "Distributed" => UUID("8ba89e20-285c-5b6f-9357-94700520ee1b"), "TimerOutputs" => UUID("a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"), "AlgebraOfGraphics" => UUID("cbdf2221-f076-402e-a563-3d30da359d67")…), Dict{String, Base.UUID}(), 
Dict{String, Vector{String}}(), Dict{String, Pkg.Types.Compat}())
```
that bit works, but if I do a `.name` as in the [code](https://github.com/JuliaDynamics/DrWatson.jl/blob/28057cec9331a17965e0fbad63cf9b26d937e5ab/src/project_setup.jl#L54) I get a `Nothing`.

One hint could be, that I noticed some dynamic/linking/missmatch in the paths on the cluster e.g. `/home/st/XXX/XXX/projects/LMMPerm` seems to be `/pfs/data5/home/st/XXX/XXX/projects/LMMPerm` in julia. Not sure whats going on :shrug:

**Minimal Working Example**
```
using DrWatson
@quickactivate("LMMPerm")
```
DrWatson v2.9.1 & Julia 1.7.3

----
Thanks a ton for the help. This package is amazing and soooo helpful!!

**Comment by Datseris** [comment]
BTW, thanks a lot for your kind words, and for reporting an excellent bug report! :)

**Comment by palday** [comment]
As a stopgap, you can fix this by adding a 

```toml
name = "YourFolderName"
```

to the top of the relevant Project.toml file. 

I think there might be something with how the project name is discovered from the environment. But maybe @behinger can contribute a docs PR :wink:

**Comment by behinger** [comment]
finally: it was not only the missing name in `Project.toml`; but also Slurmm moved the file to a temporary location, thus `@__dir__` was not equal to the `pwd()` in my case. Now I use `quickactivate(pwd(),"LMMPerm")` and it seems to work
